exercises/verify.cpp

```diff
@@ -6,13 +6,23 @@ Insert number 3
 The number 3 is [not] present in the array.
 */
 
+
+#define EXIT_SUCCESS 0
 #include <iostream>
+
 using namespace std;
 
+bool isPresent(int N[], int size, int n) {
+  for(int i = 0; i<size; i++)
+    if(N[i] == n)
+      return true;
+  return false;
+}
+
 int main()
 {
   // placeholder
   int N[10] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
-
-  return 0;
+  isPresent(N, 10, 3) ? cout << "The number 3 is present in the array." << endl : cout << "The number 3 is not present in the array." << endl;
```

Comment:
questo lo potresti scrivere meglio, ciò che cambia tra una stringa e l'altra è quel "not" potresti fare tutto più compatto eliminando il codice "ridondante" così:
```C++
cout << "The number 3 is" << (isPresent(N, 10, 3) ? "" : " not")  << " present in the array." << endl;
```

Reply:
Come ho detto nell'altro esercizio non conoscevo l'uso di questa sintassi, ti ringrazio, cambio subito

Reply:
si impara molto dalle code review
ed anche leggendo il codice degli altri 👍